examples/PyOpenCL/CMakeLists.txt

```diff
@@ -38,7 +38,11 @@ if (EXISTS "${TS_BUILDDIR}/mypy/bin/py.test" AND
   set(ACTUALLY_ENABLED_TESTSUITES ${ACTUALLY_ENABLED_TESTSUITES} PARENT_SCOPE)
 
   add_test(NAME "PyOpenCL"
-           COMMAND "${TS_SRCDIR}/test_build_dir")
```

Comment:
This script keeps the status to avoid regressing on the number of passing pyopencl tests. Or do all of them now pass and the main py.test can be executed directly?